micro-ui/web/micro-ui-internals/packages/css/CHANGELOG.md

```
@@ -1,8 +1,10 @@
 # Changelog
 
-## [1.8.27=8] - [04-June-2025]
-- Added `Carousel.scss` for login screen layout.
+## [1.8.29] - [09-June-2025]
+-Override `digit-label-field-pair` class in `login-container`
 
+## [1.8.28] - [04-June-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)